src/System.Net.Sockets/src/System/Net/Sockets/TCPClient.cs

```diff
@@ -72,6 +72,22 @@ protected bool Active
 
         public int Available { get { return AvailableCore; } }
 
+        // Used by the class to provide the underlying network socket.
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)] // TODO: Remove once https://github.com/dotnet/corefx/issues/5868 is addressed.
```

Comment:
The concern is that the Unix implementation has that mitigation which changes the behavior of Connect if Client is accessed first.  That means that hovering over the Client property in the debugger could evaluate it and end up accidentally changing the behavior of Connect.  This is a mitigation for that until Debugger.NotifyOfCrossThreadDependency is exposed and we can invoke that from the Unix implementation's Client property to prevent this from happening accidentally. That said, I don't know if debuggers on Unix (e.g. VS Code) respect this.